module/Database/src/Mapper/ProspectiveMember.php

```diff
@@ -48,15 +48,17 @@ public function search(string $query): array
             ->from(ProspectiveMemberModel::class, 'm')
             ->where("CONCAT(LOWER(m.firstName), ' ', LOWER(m.lastName)) LIKE :name")
             ->orWhere("CONCAT(LOWER(m.firstName), ' ', LOWER(m.middleName), ' ', LOWER(m.lastName)) LIKE :name")
+            ->orWhere("m.email LIKE :name")
             ->setMaxResults(32)
             ->orderBy('m.lidnr', 'DESC')
             ->setFirstResult(0);
 
         $qb->setParameter(':name', '%' . strtolower($query) . '%');
 
-        // also allow searching for membership numbers
+        // also allow searching for membership or TUe numbers
         if (is_numeric($query)) {
             $qb->orWhere("m.lidnr = :nr");
+            $qb->orWhere("m.tueUsername = :nr");
```

Comment:
Same as above.

Reply:
Updated in cead09482498abd8002ed8f3db3dc981592ca435